packages/cli/src/Init.ts

```diff
@@ -156,7 +156,7 @@ export class Init implements Command {
       process.exit(1)
     }
 
-    if (fs.existsSync(prismaFolder)) {
+    if (fs.existsSync(prismaFolder) && fs.lstatSync(prismaFolder).isDirectory()) {
```

Comment:
Do we want to add an error message for when it's a file that already exists?

Reply:
Hm, is that problematic? I was assuming that an existing file should not be a problem and as such should be skipped in the check as the folder should be able to be created then.

Reply:
It might be that some filesystems possibly allow that, but generally and on most platforms, yes, that would be problematic.

```
/tmp
❯ touch prisma

/tmp
❯ mkdir prisma
mkdir: prisma: File exists
```

Reply:
Ok, that makes this PR useless indeed.

What we could do instead if be clearer in the error message if it is a file or folder - but that does not really justify a change.